graphql/README.md

```diff
@@ -32,14 +35,13 @@ Default config:
 
 ```hocon
     graphql {
-        keepAliveIntervalSeconds: 15  # interval to send keepalive messages over apollo websocket protocol in seconds
-        webSocketSubProtocol: "graphql-ws"  # apollo websocket subprotocol to identify as
-        idleTimeout: null  # allows override of jetty websocket policy idleTimeout
-        maxBinaryMessageSize: null  # allows override of jetty websocket policy maxBinaryMessageSize
-        maxTextMessageSize: null  # allows override of jetty websocket policy maxTextMessageSize
-        checkAuthorization: false  # whether to kick unauthenticated clients off websocket sessions (allow by default)
-        connectionInitWaitTimeout: 15  # when using graphql-ws protocol, must receive the connection_init message within this many seconds
-    }
+  keepAliveIntervalSeconds: 15  # interval to send keepalive messages over websocket/sse protocols in seconds
```

Comment:
was this format change on purpose?

Reply:
markdown intellij plugin did it, not sure why but didn't want to fight it